uvicorn/lifespan/on.py

```diff
@@ -1,24 +1,28 @@
 import asyncio
 import logging
+from asyncio import Queue
+
+from uvicorn import Config
+from uvicorn._types import LifespanReceiveMessage, LifespanScope, LifespanSendMessage
 
 STATE_TRANSITION_ERROR = "Got invalid state transition on lifespan protocol."
 
 
 class LifespanOn:
-    def __init__(self, config):
+    def __init__(self, config: Config) -> None:
         if not config.loaded:
             config.load()
 
         self.config = config
         self.logger = logging.getLogger("uvicorn.error")
         self.startup_event = asyncio.Event()
         self.shutdown_event = asyncio.Event()
-        self.receive_queue = asyncio.Queue()
+        self.receive_queue: Queue = asyncio.Queue()
```

Comment:
asyncio.Queue is generic, but only in the typestubs. So if you quote it you can do:
```suggestion
        self.receive_queue: "Queue[LifespanReceiveMessage]" = asyncio.Queue()
```

Reply:
Or possibly to avoid the extra/slightly redundant input:
```suggestion
        self.receive_queue: "asyncio.Queue[LifespanReceiveMessage]" = asyncio.Queue()
```

Reply:
done !